CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Contributions from the community are essential in keeping Infinispan strong and
 
 This guide focuses on how to contribute back to Infinispan using GitHub pull requests.
 If you need help with cloning, compiling or setting the project up in an IDE please refer to
-[this page](http://infinispan.org/docs/stable/contributing/contributing.html).
+[this page](https://infinispan.org/docs/stable/titles/contributing/contributing.html).
```

Comment:
what about replacing "this page" with "the contributing guide"?
`please refer to this page.` vs  `please refer to the contribution guide`
WDYT?
ps. same for the below

Reply:
yeah good call @pruivo